stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/ImageLoader.kt

```diff
@@ -122,15 +122,15 @@ public object ImageLoader {
         onComplete: () -> Unit = {},
     ) {
         coilLoadAny(data, context.streamImageLoader) {
-            placeholderResId?.let { placeholder(it) }
+            placeholderResId?.let(::placeholder)
             listener(
                 onStart = { onStart() },
                 onCancel = { onComplete() },
                 onError = { _, _ -> onComplete() },
                 onSuccess = { _, _ -> onComplete() },
             )
             if (videoContentUri) {
-                fetcher(VideoFrameUriFetcher(context))
+                fetcher(VideoFrameFileFetcher(context))
```

Comment:
I think we need both fetchers, otherwise thumbnails of videos from galley tab/files tab/file manager will not be displayed

Reply:
Looking at the implementation, I think the last `fetcher` call will be the only one that takes effect this way, because we're just setting the value of a single property inside Coil with this.

Reply:
That's bad, then we need two separate methods with different fetchers and more boilerplate

Reply:
Let's add them, and afterwards I'll try to review all of the stuff we have in `ImageLoader` because it's getting out of hand 😄

Reply:
Our `CaptureMediaContract` can return an Uri instead of a File, then it could be rendered with the old `VideoFrameUriFetcher`.
We shouldn't need to add an extra method to render them.
Or we can create an Uri from file to be rendered when we capture video/picutre, if we still need the File itself to be sent to backend

Reply:
To create a proper Uri to our SDK you need to use `StreamFileProvider.getUriForFile(context, file)`

Reply:
i'll take a look

Reply:
Not this is only using `StreamFileProvider.getUriForFile(context, file)` the URI works as it should